CONTRIBUTING.rst

```diff
@@ -55,6 +60,17 @@ method is a wrapper around the `requests.request` function that provides
 important astroquery-specific utility, including caching, HTTP header
 generation, progressbars, and local writing-to-disk.
 
+Dependencies
+------------
+New contributions are generally not allowed to bring along additional dependencies.
+
+The astropy ecosystem tools should be used whenever possible.
+For example, `astropy.table` should be used for table handling,
+`astropy.io.ascii` for ascii-table parsing, and `astropy.units` for unit and quantity
```

Comment:
I would drop the io.ascii part here, as I think many cases can be instead handled by Table itself, no need to go one layer deeper for io.fits or io.ascii.
```suggestion
or `astropy.units` for unit and quantity
```